main/lsp/LSPTypecheckerCoordinator.cc

```diff
@@ -52,41 +52,43 @@ class TypecheckerTask final : public core::lsp::Task {
 
 class InitializeTask : public LSPTask {
     LSPTypechecker &typechecker;
-    WorkerPool &workers;
```

Comment:
Now that `TypecheckerDelegate` lets the task access `workers`,it doesn't need to store it separately.